src/k8s/tests/request.test.js

```diff
@@ -300,4 +302,20 @@ describe('request.js', () => {
       expect(vm.spec.template.spec.domain.resources.requests.memory).toBe('1G');
       return vm;
     }));
+  it('no pod network', () =>
+    createVM(k8sCreate, templates, customFlavor, networks).then(vm => {
+      expect(vm.spec.template.spec.domain.devices.autoattachPodInterface).toBeFalsy();
+      expect(vm.spec.template.spec.domain.devices.interfaces).toBeUndefined();
+      expect(vm.spec.template.spec.networks).toBeUndefined();
+      return vm;
+    }));
+  it('only multus network', () =>
```

Comment:
please use more verbose descriptions, it is not obvious what it should test

Reply:
is it better now ?

Reply:
ok